### ISSUE TITLE
Make sure types are updated with new methods

### ISSUE THREAD

**Issue by filipenevola** [quavedev/oss#9]
When we are coding with Meteor 3 the new Async methods are not being detected

For example, use fetchAsync in CodeFTW Meteor-3 branch and see how the IDE behaves

**Comment by 9Morello** [comment]
This issue is caused by the outdated type definitions from WebStorm's Meteor plugin [being too old](https://github.com/JetBrains/intellij-plugins/tree/master/Meteor/resources/tsMeteorStubs). As far as I can tell, it only contain type definitions for Meteor 1.3.1, which are used regardless of what version of Meteor your project actually uses.

This plugins' type definitions seem to be handwritten and updating it seems non-trivial.

IMO the easiest way to fix this for the end user is to ask them to install install Meteor types from [DefinitelyTyped](https://www.npmjs.com/package/@types/meteor). This will get types and autocomplete properly working on WebStorm, and also on VSCode/Codium if the users isn't using the `quave:collections` collections package.

Tl;DR run `meteor npm install @types/meteor` and autocomplete will work properly with updated type definitions
